gulpfile.js

```diff
@@ -41,8 +43,16 @@ gulp.task('webpack-build', done => {
   });
 });
 
+gulp.task('update-qext-version', function () {
+  return gulp.src("./build/" + name + ".qext")
```

Comment:
need to use buildDestination from settings, otherwise it won't work when BUILD_PATH is set

Reply:
I'm on it!